.changeset/slow-tips-jam.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
This should be patch so that it gets published

Reply:
Fixed